rotary/rotary.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python
+
+"""
+Created by: Lee Bergstrand (2023)
+
+Description: A command-line tool for the Rotary hybrid assembly workflow.
```

Comment:
Change as follows?
"A command-line interface for the Rotary genomics toolkit"

Reply:
Not sure if "interface" is the best word here -- thoughts? I didn't want to repeat "tool" twice.

Reply:
@jmtsuji Addressed in https://github.com/rotary-genomics/rotary/pull/102